src/vs/workbench/contrib/comments/browser/commentsView.ts

```diff
@@ -287,6 +302,9 @@ export class CommentsPanel extends FilterViewPane implements ICommentsView {
 	private createMessageBox(parent: HTMLElement): void {
 		this.messageBoxContainer = dom.append(parent, dom.$('.message-box-container'));
 		this.messageBoxContainer.setAttribute('tabIndex', '0');
+		const focusTracker = this._register(dom.trackFocus(this.messageBoxContainer));
+		this._register(this._onDidFocusResults.add(focusTracker.onDidFocus));
+		this._register(this._onDidBlurResults.add(focusTracker.onDidBlur));
```

Comment:
This one is trickier to untangle, but should still be possible to do without any additional allocations. Let me know if you need pointers for that.